src/Assert.php

```diff
@@ -1462,11 +1458,9 @@ public static function file($value, $message = '')
      */
     public static function directory($value, $message = '')
     {
-        static::fileExists($value, $message);
```

Comment:
This is a pointless check. If the directory does not exist, the next assertion will never run. If the directory does exist, then it duplicates work.